corehq/apps/users/models.py

```diff
@@ -2683,6 +2683,9 @@ class Invitation(models.Model):
     role = models.CharField(max_length=100, null=True)  # role qualified ID
     program = models.CharField(max_length=126, null=True)   # couch id of a Program
     supply_point = models.CharField(max_length=126, null=True)  # couch id of a Location
+    location_id = models.CharField(max_length=126, null=True)  # couch id of a Location (to replace supply_point)
+    profile = models.ForeignKey("custom_data_fields.CustomDataFieldsProfile",
```

Comment:
I see you added `profile` to the model, but there's nothing to store custom user data that isn't part of the profile.  What are you thinking of doing for that stuff?  Could probably hold it in a dict or as a FK reference to a `SQLUserData` object, though that latter might be tricky without a user to attach it to.